### ISSUE TITLE
[Bug]: Scrolling the Category list in the right-click menu constantly readjusts the list's width up or down to the widest element currently shown

### ISSUE THREAD

**Issue by Nocifer** [VueTorrent/VueTorrent#1987]
### Bug description

The list begins with a width equal to the width of the widest element among the first 15-20 or so categories (the exact amount that can be displayed at any one time in the list depends on the zoom level, resolution, scaling factor, etc) plus the "Create new category" item at the top - in my case, that's the widest element initially.

Scrolling down the list, when that initially widest element disappears, or if a wider element appears, the list will immediately readjusts itself to the width of the currently widest element.

### Expectations

The list should have a fixed width, probably adjusted to the widest element of the whole list, regardless of if that element is currently being displayed or not (i.e. the proper width should be pre-evaluated).

### Relevant debug informations

N/A

### Steps to reproduce

1. Create a bunch of categories of varying title sizes ( e.g. 'test', 'teeest', 'teeeeeest', 'teeeeeeeeeeeeeest', etc), enough of them to need to scroll the category list in order to view them them all (so probably at least 15-20).
2. Scroll up and down the list and observe how the width constantly changes.

### How did you install VueTorrent?

Downloaded from GitHub releases

### Environment description

```plain text
Device type: Desktop
OS: Arch Linux
Browser (name and version): Firefox 131, Chromium 130

PWA: No
Running inside a container: qB yes, Vuetorrent no
```

### qBittorrent version

4.6.5

### VueTorrent version

2.15.0

### Agreements

- [X] The bug still occurs on the nightly build
- [X] The bug still occurs on the latest version of qBittorrent
- [X] A similar issue doesn't already exist
- [X] I checked the DevTools and included any errors that may show up

**Comment by Larsluph** [comment]
This is a Vuetify limitation to allow for an infinite list of items to be shown, because items aren't loaded all at once for performance and memory usage reasons.

We could approximate the max width depending on the size of the longest element but we need to consider the computation needed to do that.

**Comment by Nocifer** [comment]
Perhaps the minimum width could be set to the "Create new category" item, which is a) already very prominent as the "title" of the list and b) wide enough to be a good default, and then only increase the list's width if a wider element appears, and reduce it back to the default/minimum when that element is removed?

**Comment by Nocifer** [comment]
If I'm not mistaken, this one has been fixed as of the latest VueTorrent release.

**Comment by Larsluph** [comment]
I think it has been fixed by #1985
Released in 2.16.0
